src/app/itemsOnLifecycle.clj

```diff
@@ -161,4 +161,25 @@
 ; the param :onlyIncorrect
 ; Either way it produces a table with two columns: class and problem. Problem can be potentially empty.
 (def utilsClassesOp
-  (Operation. utilsClassesQuery [:onlyIncorrect] [:class :problem]))
\ No newline at end of file
+  (Operation. utilsClassesQuery [:onlyIncorrect] [:class :problem]))
+
+; ============================================
+; ITEM 7
+; ============================================
+
+(defn calls-finalizers? [class]
+  (pos? (count
+          (filter #(and (= "finalize" (.getName %))
```

Comment:
Checking for `getArgs` to be not nil is correct for how JavaParser works, however in the next release it is going to return an empty list so I will update the check for checking for nil or empty list.